### ISSUE TITLE
Bad err msg and behaviour when Pipeline Name already exists.

### ISSUE THREAD

**Issue by eea71** [OpenDataNode/open-data-node#131]
I tried to create pipeline and I used NAME already existing pipeline. Internal catalog rised error and not stayed at the present page, but pushed me at the next page. Button BACK returned me at the position when all pipelines are listed. 
![image](https://cloud.githubusercontent.com/assets/10993265/7836603/a9fe77c8-0481-11e5-98e7-f87909bb383c.png)

Right behaviour is to rise err msg like: Pipeline name is already in use, please input other name..
Or something like this. And stay at the same page.

**Comment by skrchnavy** [comment]
depends on OpenDataNode/ckanext-odn-pipeline#3

**Comment by skrchnavy** [comment]
OpenDataNode/ckanext-odn-pipeline#3 closed, so verifcation needed

**Comment by eea71** [comment]
Without change...

**Comment by mvi-eea-sk** [comment]
@skrchnavy That's why i changed the state to "in progress"

Most of the changes i have already done here: https://github.com/OpenDataNode/ckanext-odn-pipeline/commit/e956add90795ed6098321b9bb13c77b2dd900708

Currently, I am waiting for changes in UV API that will make it so the calls will return proper error messages for me to display them to user. Otherwise I can only display some general error message,  guess what really happend or display some horrific messages describing DB errors:
```
Could not commit JPA transaction; nested exception is javax.persistence.RollbackException: Exception [EclipseLink-4002] (Eclipse Persistence Services - 2.5.1.v20130918-f2b9fc5): org.eclipse.persistence.exceptions.DatabaseException
Internal Exception: org.postgresql.util.PSQLException: ERROR: duplicate key value violates unique constraint "ppl_model_name_key"
Error Code: 0
Call: INSERT INTO ppl_model (ID, DESCRIPTION, last_change, NAME, visibility, user_actor_id, user_id) VALUES (?, ?, ?, ?, ?, ?, ?)
	bind => [41, testing errors, 2015-06-11 11:07:23.178, relational from CKAN to CKAN, 0, null, 1]
Query: InsertObjectQuery(cz.cuni.mff.xrg.odcs.commons.app.pipeline.Pipeline@2d0)
```

Until then I can't merge this feature... so still it's in progress

**Comment by mvi-eea-sk** [comment]
related to https://github.com/OpenDataNode/ckanext-odn-pipeline/issues/4

**Comment by eea71** [comment]
in fix 1.1.2 seem si ok. ;-)
